mythril_multiboot2/src/services.rs

```diff
@@ -37,4 +37,35 @@ impl VmServices for Multiboot2Services {
     fn acpi_addr(&self) -> Result<HostPhysAddr> {
         Ok(HostPhysAddr::new(0))
     }
+
+    fn rsdp(&self) -> Result<crate::acpi::rsdp::RSDP> {
+        // Panic if can find a multiboot rsdpv2 tag
+        match self.info.rsdp_v2_tag() {
```

Comment:
The tags are not exclusive, so I think there is no reason to panic just because we found a v2 tag. We can just ignore it.